api/views.py

```diff
@@ -120,3 +120,11 @@ class AssetTypeViewSet(ModelViewSet):
     permission_classes = [IsAuthenticated, ]
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ['get', 'post']
+
+
+class AssetModelNumberViewSet(ModelViewSet):
+    serializer_class = AssetModelNumberSerializer
+    queryset = AssetModelNumber.objects.all()
+    permission_classes = [IsAuthenticated, ]
+    authentication_classes = (FirebaseTokenAuthentication,)
+    http_method_names = ['get', 'post']
```

Comment:
does the `get` return the `make_label ` label or id?

Reply:
It returns the `id`

Reply:
can we return the label? we should probably do the same for the others as well (for get and detail)

Reply:
on it